chiselFrontend/src/main/scala/chisel3/Data.scala

```diff
@@ -265,6 +259,7 @@ object Flipped {
   * @define coll data
   */
 abstract class Data extends HasId with NamedComponent with SourceInfoDoc { // scalastyle:ignore number.of.methods
+
   // This is a bad API that punches through object boundaries.
   @deprecated("pending removal once all instances replaced", "chisel3")
```

Comment:
Maybe we could refactor this out? Perhaps in a separate PR?

Reply:
Yeah, this shouldn't be in this commit. Good catch.

Reply:
Fixed in rebase.